### ISSUE TITLE
Attributes and tags and multiline patterns should allow blank lines before them

### ISSUE THREAD

**Issue by stasm** [projectfluent/python-fluent#15]
Currently, none of the following messages parses correctly:

```
foo =


    Mutliline Foo Value


bar1


    .attr = Attr


bar2


    .attr1 = Attr1


    .attr2 = Attr2



bar3 =


    Multiline Bar Value


    .attr = Attr


bar4


    .attr =


        Multiline Attr Value


qux1 = Qux


    #tag


qux2 = Qux


    #tag1


    #tag2
```

**Comment by stasm** [comment]
@zbraniecki I'd love to get your help on this when you're back.

**Comment by zbraniecki** [comment]
ugh... I would love us not to allow for that tbh. It looks very confusing to read and the only cases where I can see anything like that ending up in the code is by mistake.
Let's error on mistake.

@mathjazz , @Pike - thoughts?

**Comment by Pike** [comment]
I could see a use in cases where you have quite a lengthy value in mutliple lines, and then a tag. Like

```
foo = 
    This is just a looong
    piece of text
    over multiple
    lines.
    #longtext
```

would probably be easier on the eye with an empty line.

**Comment by stasm** [comment]
I agree with @Pike. I would also like us to err on the side of least surprise. Throwing a syntax error because of a blank line would be surprising to me.

**Comment by zbraniecki** [comment]
ok. I'm convinced by your arguments.

**Comment by stasm** [comment]
@zbraniecki We need to fix the parsers to actually allow the blank lines :)

**Comment by zbraniecki** [comment]
ok. I'm convinced by your argument.

**Comment by zbraniecki** [comment]
this has been fixed in #20
